.gitmodules

```diff
@@ -61,3 +61,6 @@
 [submodule "cmake/external/onnx-tensorrt"]
 	path = cmake/external/onnx-tensorrt
 	url = https://github.com/stevenlix/onnx-tensorrt.git
+[submodule "cmake/external/rknpu_ddk"]
```

Comment:
I took a look at https://github.com/airockchip/rknpu_ddk. It seems like the main purpose is for distributing the rknpu library?
it's preferred to not add this as a submodule to onnxruntime repo. We don't users to pull binaries when cloning onnxruntime repo.
instead, can you list these libraries as a dependency in the BUILD instructions and direct users to download the libraries themselves?

Reply:
ok, I have moved rknpu_ddk out of cmake/external.